### ISSUE TITLE
BlurView doesn't seem to work when using Nativescript-Vue

### ISSUE THREAD

**Issue by dangrima90** [Akylas/nativescript-blurview#1]
### Make sure to check the demo app(s) for sample usage 
Unfortunately I couldn't get the demo to work - both for Android and iOS. I tried running the demo app using Nativescript 6.

When trying to build for both I got the following error:

```bash
ERROR in /Users/gria001/Downloads/ui-blurview-master/node_modules/@types/node/ts3.6/globals.global.d.ts
ERROR in /Users/gria001/Downloads/ui-blurview-master/node_modules/@types/node/ts3.6/globals.global.d.ts(1,13):
TS2403: Subsequent variable declarations must have the same type.  Variable 'global' must be of type 'Global', but here has type 'Global & typeof globalThis'.
```

Been trying to use this library with Nativescript-Vue like so:

`main.js`

```js
// Register Element
Vue.registerElement(
  "BlurView",
  () => require("@nativescript-community/ui-blurview").BlurView
);
```

**Using the component**

```vue
<GridLayout>
  <Image
    width="100"
    height="100"
    src="~/assets/images/NativeScript-Vue.png"
  />
  <BlurView :blurRadius="10" />
</GridLayout>
```

### Which platform(s) does your issue occur on?
I've tested this on both Android and iOS emulators.

### Please, provide the following version numbers that your issue occurs with:

- CLI: 7.2.1
- Cross-platform modules: 7.0.0

### Please, tell us how to recreate the issue in as much detail as possible. 
Describe the steps to reproduce it.

Simply try to use the component. In the following screenshots I've added a border to the `BlurView` so that I could check whether it was being rendered or not.

Result on Android: No changes whatsoever

![image](https://user-images.githubusercontent.com/36479149/131361239-ac9b6487-f48c-4012-849c-575184c39916.png)

Result on iOS: For some reason image is converted to greyscale

![image](https://user-images.githubusercontent.com/36479149/131361207-2ccb977f-b313-48ec-8156-eb2b66d5eac8.png)

In the meantime I'm going to try to get the demo to work maybe I can better see what the issue might be.

**Comment by farfromrefug** [comment]
@dangrima90 did you try to force a size to your blurview? There is no autosize  with blurview

**Comment by dangrima90** [comment]
@farfromrefug you are right, having a width and height does solve the issue. Works perfectly on Android, however iOS still results in a greyscale image.

**Updated code**

```vue
<GridLayout>
  <Image
    width="100"
    height="100"
    src="~/assets/images/NativeScript-Vue.png"
  />
  <BlurView width="100%" height="100" :blurRadius="10" />
</GridLayout>
```

**Android**

![image](https://user-images.githubusercontent.com/36479149/131504961-c6bb2d32-7770-4712-b50e-938d41275012.png)

**iOS**

![image](https://user-images.githubusercontent.com/36479149/131504997-b3f5d1e8-ac6b-4543-80d5-5d9df5c04af8.png)
